README.md

```diff
@@ -20,20 +20,22 @@ Jump in: [![Slack Status](http://slack.projecthydra.org/badge.svg)](http://slack
   * [What is Hyrax?](#what-is-hyrax)
     * [Feature list](#feature-list)
   * [Help](#help)
-  * [Getting started](#getting-started)
+  * [Getting Started](#getting-started)
     * [Prerequisites](#prerequisites)
-      * [Characterization](#characterization)
-      * [Derivatives](#derivatives)
+      * [FITS](#fits)
```

Comment:
Can we not rename these headers to reference specific tools (which we may change in the future)? This way any documentation elsewhere that links into the `#characterization` or `derivatives` anchors won't break.